source/_drafts/Video-js-7-Roadmap.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Video.js 7 Roadmap
+author:
+  name: Gary Katsevman
+  github: gkatsev
+tags:
+  - IE8
+  - IE
+  - google analytics
+  - VHS
+  - webpack
+---
+![](woodland-road-falling-leaf-natural-38537.jpeg)
+
+Today, I’m excited to announce the roadmap to Video.js 7! While this is a major version update, the actual changes are not that major. The two main changes are the addition of videojs-http-streaming, VHS for short, and the removal of support for older versions of Internet Explorer.
+
+![VJS 7](vjs 7 small.png)
+
+## Video.js HTTP Streaming
+
+![](VHS small.png)
+
+VHS used to be videojs-contrib-hls, but as MPEG-DASH became more popular, we realized that we wanted to support it as well and that we can share a lot of the code between HLS and DASH implementations. Look for a more details in an announcement post for VHS soon!
+VHS will ship with Video.js by default because one of the guiding principles of Video.js is to make it easy for users to just put it on the page and have a player that works across browsers. With HLS and DASH becoming so popular, we thought it was about time for a plug and playable Video.js for the most common streaming formats.
+In addition to having it be included by default, we’ll make sure to provide builds that exclude VHS for those that don’t need it or are using another playback engine like HLS.js.
+
+## Old IE
+
+![IE8](IE8 small.png)
+
+Video.js has a long history of supporting IE. It has a flash fallback that was originally created for IE8. When Video.js 5 was released, we had a lot of code in place to support IE8 and when Video.js 6 came around, with Flash on its way out, we moved Flash support to a separate project, videojs-flash. Now, based on usage data we have, we are planning to remove support for IE8, IE9, and IE10.
```

Comment:
I might reword this first sentence a little. How do you feel about:

> Video.js has a long history of making a best effort to support IE, starting with a Flash fallback originally created for IE8.

The last sentence feels a little ominous. What about just:

> Now, based on usage data, we are planning ...